dotCMS/src/integration-test/java/com/dotcms/contenttype/test/ContentTypeAPIImplTest.java

```diff
@@ -101,6 +91,47 @@ public void testFieldsMethod() throws Exception {
 
 	}
 
+	// Based on: https://groups.google.com/forum/?pli=1#!topic/dotcms/2-0QrRJtppw
+	@Test
+	public void Test_Fields_without_contenttype_on_saving() throws Exception {
+
+		ContentTypeAPI ctApi = APILocator.getContentTypeAPI(APILocator.systemUser());
+		ImmutableSimpleContentType.Builder builder = ImmutableSimpleContentType.builder();
+		ContentType movie = builder.name("Movie").folder(APILocator.systemHost().getFolder()).build();
+
+		movie = ctApi.save(movie);
```

Comment:
should you remove this save for the test?

Reply:
on it